SimPEG/EM/Static/DC/ProblemDC.py

```diff
@@ -305,6 +438,10 @@ class Problem3D_N(BaseDCProblem):
 
     def __init__(self, mesh, **kwargs):
         BaseDCProblem.__init__(self, mesh, **kwargs)
+        # Not sure why I need to do this
+        # To evaluate mesh.aveE2CC, this is required....
```

Comment:
what is the question here??

Reply:
Oh, this was a problem with `TREE` mesh, I had to evaluate `nodalGrad` first, if I want to obtain aveE2CC. Any ideas @rowanc1 ?

Reply:
It should just be that the Mesh has to be numbered. I believe there is a function that does that specifically `mesh.number()` (maybe?) if that is already done, it will return fast.